### PR TITLE
Fixup a couple of code example issues

### PR DESCRIPTION
**Pull request by nwbirnie**
Change-Id: Id69d2649fa08260c9ce0f95c06747936c13b19b2

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java [hunk 1]
```diff
@@ -600,14 +601,20 @@ private MutateOperation createAdGroupAdOperation(
...
+          ++i) {
         adBuilder
             .getSmartCampaignAdBuilder()
             .addHeadlines(AdTextAsset.newBuilder().setText("Placeholder headline " + i).build());
       }
     }
     if (adSuggestions.getDescriptionsCount() < NUM_REQUIRED_DESCRIPTIONS) {
-      for (int i = 0; i < NUM_REQUIRED_DESCRIPTIONS - adSuggestions.getDescriptionsCount(); ++i) {
+      for (int i = 0;
+          i
```
Comment (fiboknacky) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java:614]: The format is a bit strange?

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java [hunk 2]
```diff
@@ -630,19 +637,39 @@ private MutateOperation createAdGroupAdOperation(
...
+    List<MutateOperation> keywordThemeOperations =
+        keywordThemeInfos.stream()
+            .map(
+                keywordTheme -> {
+                  MutateOperation.Builder builder = MutateOperation.newBuilder();
+                  builder
+                      .getCampaignCriterionOperationBuilder()
+                      .getCreateBuilder()
+                      .setCampaign(ResourceNames.campaign(customerId, SMART_CAMPAIGN_TEMPORARY_ID))
+                      .setType(CriterionType.KEYWORD_THEME)
```
Comment (fiboknacky) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java:652]: I think this is not necessary in the golden CL?

**Review by fiboknacky — CHANGES_REQUESTED** [review]

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java (hunk 1 above)
Comment (nwbirnie) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java:614]: I know - I checked before and rechecked now, this is google-java-format!

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java (hunk 2 above)
Comment (fiboknacky) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java]: What about this?

**Review by fiboknacky — APPROVED** [review]

**Review by AnashOommen — APPROVED** [review]

**Pull request merged**